squad/frontend/views.py: Add support for filtering project list by age

katieworton opened the pull request:
Add a group parameter for filtering projects in a group home their age as an alternative to limiting by count.

katieworton commented:
Drafted code - need to add unit testing.

chaws commented on squad/frontend/views.py:
```diff
@@ -99,8 +100,19 @@ def group_home(request, group_slug):
-    display_all_projects = request.GET.get('all_projects') is not None
     num_projects = group.get_setting('DEFAULT_PROJECT_COUNT')
+
+    max_project_age_days = group.get_setting('MAX_PROJECT_AGE_DAYS')
```
I thought it over, and what do you think about the name `SHOW_PROJECTS_ACTIVE_N_DAYS_AGO`?

chaws commented:
Just one comment about the variable name, but LGTM!

katieworton commented:
@chaws I have added unit tests and refactored things a bit (instead of passing `request` to the function, I pass in `user`, `order_by` and `display_all_projects` - makes a lot more sense to do it like this & made it easier to write the unit tests).

Let me know what you think!

chaws reviewed (approved):
LGTM! I think your tests covered all possible scenarios for this feature
